gdal/ogr/ogrsf_frmts/geomedia/ogrgeomediadatasource.cpp

```diff
@@ -140,6 +140,28 @@ int OGRGeomediaDataSource::Open( const char * pszNewName, int bUpdate,
         }
     }
 
+    // check for GAliasTable table
+    bool bFoundGAliasTable = false;
+    {
+        CPLODBCStatement oTableList( &oSession );
+        if( oTableList.GetTables() )
+        {
+            while( oTableList.Fetch() )
+            {
+                const CPLString osTableName = CPLString( oTableList.GetColData(2) );
+                const CPLString osLCTableName(CPLString(osTableName).tolower());
+
+                if( !osLCTableName.empty() && osLCTableName == "galiastable" )
```

Comment:
can be likely simplified as (was the empty() check to be slighly more performant ? but in that case I believe that empty string is highly unlikely, so that will not be necessary)
```suggestion
                if( osLCTableName == "galiastable" )
```